cmd/jaeger/internal/extension/jaegerstorage/extension_test.go

```diff
@@ -220,7 +220,7 @@ func makeStorageExtenion(t *testing.T, config *Config) component.Component {
 	return storageExtension
 }
 
-func startStorageExtension(t *testing.T, memstoreName string) component.Component {
+func StartStorageExtension(t *testing.T, memstoreName string) component.Component {
```

Comment:
is this change necessary? I don't see this function being called from outside of this package (and if it were it would be a code smell, we don't share test code like that)